certbot/tests/main_test.py

```diff
@@ -593,6 +594,17 @@ def _call(self):
             mock_storage.return_value = self.account_storage
             return _determine_account(self.config)
 
+    @mock.patch('certbot._internal.client.register')
+    @mock.patch('certbot._internal.client.display_ops.get_email')
+    def _register_error_base(self, err_msg, exception, mock_get_email, mock_register):
```

Comment:
To follow existing conventions:

```suggestion
    def _register_error_common(self, err_msg, exception, mock_get_email, mock_register):
```